### DISCUSSION TITLE
Ublock classifies many well-known open source software sites as badware risk ?

### DISCUSSION THREAD

**Discussion by LucMtl63** [uBlockOrigin/uAssets#21473] [General]
Hello, long term user of Ublock here, I'm surprised to find that recently Ublock started to block access to many well-known open source software sites, calling them badware risks.

Here's some current extract from ublock-badware  :  

||audacity.de^$doc
||7zip.es^$doc
||7zip.fr^$doc
||7zip.it^$doc
||adblock.fr^$doc
||audacity.es^$doc
||audacity.fr^$doc
||audacity.it^$doc
||audacity.pl^$doc
||blender3d.es^$doc
||blender3d.fr^$doc
||celestia.es^$doc
||celestia.fr^$doc
||freefilesync.com^$doc
||gimp.es^$doc
||inkscape.es^$doc
||inkscape.fr^$doc
||inkscape.it^$doc
||scribus.fr^$doc
||scribus.it^$doc
||stellarium.es^$doc
||stellarium.fr^$doc

I found out about it trying to access scribus.fr

What's going on ?

Thanks.

**Comment by stephenhawk8054** [comment]
`scribus.net` is the official site. `scribus.fr` is not.

@MasterKia @Yuki2718 But I think we should not block unofficial site if they use official site's links to download? We should only block the ones that use other hosts for downloading.

**Reply by LucMtl63** [reply]
Yes, or maybe just blocking the other hosts download links.  These pages have no malicious intent, and they provide information about the software in languages not necessarily supported by the official site, which is the case for Scribus for example.  

Thanks for your answer.

**Reply by MasterKia** [reply]
> But I think we should not block unofficial site if they use official site's links to download? We should only block the ones that use other hosts for downloading.

We can't keep monitoring whether they'll one day decide to change the links and distribute malware.
This has happened before with [`revanced.io`](https://github.com/uBlockOrigin/uAssets/pull/17469) , [`revancedextended.com`](https://github.com/uBlockOrigin/uAssets/issues/18857#issuecomment-1625235048) and Fitgirl repacks:
![image](https://github.com/uBlockOrigin/uBlock-issues/assets/17685483/0d0f6a05-7ff8-4006-8fbf-28fd9aaaa410)

Ideally there should be a big disclaimer on top of the site to let the user know that this site is not official.

***
The block for `scribus.fr` is `$doc`, so users can still proceed to the site if wanted.
For sites that point to the official links, maybe we could use `$doc` and for other sites we'd use `$all`.

**Reply by stephenhawk8054** [reply]
We don't need to monitor the sites. I think it's better to block when there are reports of them. That way we can also have issues references for future as well. Otherwise, I don't know how to respond to the complaints when the references are about other sites/services.

**Reply by Yuki2718** [reply]
> These pages have no malicious intent,

Happy if you or anyone investigate which ones don't and thus can be removed from the list, as some of them indeed distribute malware as shown in the [VT link](https://www.virustotal.com/#/file/a5616985e92ca7c1df3b132d2da2ef33c64f38ba2dca40445017037473d7d014/detection) in the comment.
